examples/README.md

```diff
@@ -0,0 +1,53 @@
+gRPC in 3 minutes (Go)
+======================
+
+BACKGROUND
+-------------
+For this sample, we've already generated the server and client stubs from [helloworld.proto](https://github.com/grpc/grpc-common/blob/master/protos/helloworld.proto). 
+
+PREREQUISITES
+-------------
+
+- This requires Go 1.4
```

Comment:
Uh, hopefully we're not going to immediately start requiring Go 1.5 for gRPC. It's polite to wait for a little while after a release before assuming everyone has upgraded.